Source/WebGPU/WebGPU/ComputePipeline.mm

```diff
@@ -49,8 +49,6 @@
     computePipelineDescriptor.maxTotalThreadsPerThreadgroup = size.width * size.height * size.depth;
     for (size_t i = 0; i < pipelineLayout.numberOfBindGroupLayouts(); ++i)
         computePipelineDescriptor.buffers[i].mutability = MTLMutabilityImmutable; // Argument buffers are always immutable in WebGPU.
-    // FIXME: https://bugs.webkit.org/show_bug.cgi?id=249345 don't unconditionally set this to YES
-    computePipelineDescriptor.supportIndirectCommandBuffers = YES;
```

Comment:
Should we close that bug?

Reply:
Not yet since it still applies to render pipelines too